<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Top 30 Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,1" columnstretch="1,0">
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_create">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_add_clip">
          <property name="text">
           <string>Add Clip</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_delete_clip">
          <property name="text">
           <string>Delete Clip</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="ClipListView" name="clip_view">
        <property name="lineWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
        <item>
         <widget class="QPushButton" name="btn_move_up">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Move Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_move_down">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Move Down</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_song_length">
            <property name="text">
             <string>Song Length:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txt_song_length">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_voice_start">
            <property name="text">
             <string>Voice Start Overlap:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txt_voice_start">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbl_voice_end">
            <property name="text">
             <string>Voice End Overlap:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="txt_voice_end">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_new"/>
    <addaction name="separator"/>
    <addaction name="act_create_clip"/>
    <addaction name="separator"/>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="act_add_clip"/>
    <addaction name="act_delete_clip"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="act_help"/>
    <addaction name="separator"/>
    <addaction name="act_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_new">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="act_create_clip">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="act_add_clip">
   <property name="text">
    <string>Add Clip...</string>
   </property>
  </action>
  <action name="act_delete_clip">
   <property name="text">
    <string>Delete Clip..</string>
   </property>
  </action>
  <action name="act_help">
   <property name="text">
    <string>Top 30 Creator Help</string>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClipListView</class>
   <extends>QTableView</extends>
   <header location="global">top30.clip_list</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
